sync.cmd

```diff
@@ -50,8 +51,8 @@ if [%src%] == [true] (
 
 if [%packages%] == [true] (
   echo Restoring all packages ...
-  echo msbuild.exe %~dp0build.proj /t:BatchRestorePackages /nologo /v:minimal /p:RestoreDuringBuild=true /flp:v=diag;Append;LogFile=%synclog% >> %synclog%
-  call msbuild.exe %~dp0build.proj /t:BatchRestorePackages /nologo /v:minimal /p:RestoreDuringBuild=true /flp:v=diag;Append;LogFile=%synclog%
+  echo msbuild.exe %~dp0build.proj /t:BatchRestorePackages /nologo /v:minimal /p:RestoreDuringBuild=true /flp:v=detailed;Append;LogFile=%synclog% >> %synclog%
+  call msbuild.exe %~dp0build.proj /t:BatchRestorePackages /nologo /v:minimal /p:RestoreDuringBuild=true /flp:v=detailed;Append;LogFile=%synclog%
```

Comment:
Using detailed verbosity is consistent with clean.sh/cmd.  I think it's a better default because it keeps the log smaller, and if it's insufficient for debugging purposes, users can manually run it with diagnostic verbosity.